meson.build

```diff
@@ -203,3 +208,22 @@ top_earlgrey = declare_dependency(
 )
 
 subdir('sw')
+
+# Write environment file
+prog_meson_write_env = meson.source_root() / 'util/meson_write_env.py'
+run_command(
+  prog_python,
+  prog_meson_write_env,
+
+  '--out-file',
+  '.env',
```

Comment:
Can we put this in `build-out`?

Reply:
:+1: I did go in circles with this one :-)